pages/topics/client-distribution.mdx

```diff
@@ -325,7 +327,7 @@ Native modules are versioned uniquely per host version.
 | required_modules      | array[string]                                                                 | The names of the native modules that the client requires |
 | metadata_version? ^1^ | ?integer                                                                      | The version of the manifest metadata                     |
 
-^1^ This field is only provided via the above rehosted updates endpoint. If this field is `null`, clients are expected to generate it themselves.
+^1^ This field is only provided via the above rehosted updates endpoint, and requires `install_id` and `platform_version` to be set and supported. If this field is `null`, clients are expected to generate it themselves.
 {/* TODO: How is it generated? */}
```

Comment:
```suggestion
^1^ This field is only provided via the above rehosted updates endpoint, and requires `install_id` and `platform_version` to be set and supported.
```
I think we just ignore this part for now. Also, is it still only available on the updates.discord.com endpoint and not the main API?

Reply:
Yep